recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 0
+  number: 1000
```

Comment:
@conda-forge/core is setting to 1000 here expected?

Reply:
Yep. The bot migration will flip that back to 0 once all the recipes are migrated.